### DISCUSSION TITLE
How do I make a character smaller?

### DISCUSSION THREAD

**Discussion by theresaway** [Leather128/LeathersFunkinEngine#146] [Q&A] [answered]
I made a character and he's too big lol please help

**Comment by SrtHero278** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/93018286/150037421-112f3370-d5d9-4ee7-b3c3-7b780249ac9f.png)
Maybe use this?

**Reply by theresaway** [reply]
where does it go?

**Reply by JordanSantiagoYT** [reply]
idk where it is

**Reply by MemTheBun** [reply]
Big Brain Time
i used it to make char BIGGER

**Reply by MemTheBun** [reply]
"graphicsSize": 2,
